### PR TITLE
Update Data Platform

### PR DESCRIPTION
**Pull request by lcaggio**
Update Data Platform foundation:
 - Migrate to Composer 2
 - Remove Playground project

### CODE REVIEW

File: blueprints/data-solutions/data-platform-foundations/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The approach adapts to different high-level requirements:
 - least privilege principle
 - rely on service account impersonation
 
-The code in this blueprint doesn't address Organization-level configurations (Organization policy, VPC-SC, centralized logs). We expect those elements to be managed by automation stages external to this script like those in [FAST](../../../fast).
+The code in this blueprint doesn't address Organization-level configurations (Organization policy, VPC-SC, centralized logs). We expect those elements to be managed by automation stages external to this script like those in [FAST](../../../fast) and this blueprint deployed on top of them as one of the [stage](../../../fast/stages/03-data-platform/dev/README.md).
```
Comment (ludoo) [inline blueprints/data-solutions/data-platform-foundations/README.md:24]: as one of the stage**s**

File: blueprints/data-solutions/data-platform-foundations/README.md [hunk 2]
```diff
@@ -39,14 +39,13 @@ This separation into projects allows adhering to the least-privilege principle b
...
-- **Load** Used to load data from the drop off zone to the data warehouse. The load is made with minimal to zero transformation logic (mainly `cast`). Anonymization or tokenization of Personally Identifiable Information (PII) can be implemented here or in the transformation stage, depending on your requirements. The use of [Cloud Dataflow templates](https://cloud.google.com/dataflow/docs/concepts/dataflow-templates) is recommended.
+- **Load** Used to load data from the drop off zone to the data warehouse. The load is made with minimal to zero transformation logic (mainly `cast`). Anonymization or tokenization of Personally Identifiable Information (PII) can be implemented here or in the transformation stage, depending on your requirements. The use of [Cloud Dataflow templates](https://cloud.google.com/dataflow/docs/concepts/dataflow-templates) is recommended. When you need to handle workloads from different teams, if strong role separation is needed between them, we suggests to customize the scirpt and have separate `Load` projects.
```
Comment (ludoo) [inline blueprints/data-solutions/data-platform-foundations/README.md:42]: we suggest (no s)

File: blueprints/data-solutions/data-platform-foundations/README.md [hunk 3]
```diff
@@ -39,14 +39,13 @@ This separation into projects allows adhering to the least-privilege principle b
...
 - **Orchestration** Used to host Cloud Composer, which orchestrates all tasks that move data across layers.
-- **Transformation** Used to move data between Data Warehouse layers. We strongly suggest relying on BigQuery Engine to perform the transformations. If BigQuery doesn't have the features needed to perform your transformations, you can use Cloud Dataflow with [Cloud Dataflow templates](https://cloud.google.com/dataflow/docs/concepts/dataflow-templates). This stage can also optionally  anonymize or tokenize PII.
+- **Transformation** Used to move data between Data Warehouse layers. We strongly suggest relying on BigQuery Engine to perform the transformations. If BigQuery doesn't have the features needed to perform your transformations, you can use Cloud Dataflow with [Cloud Dataflow templates](https://cloud.google.com/dataflow/docs/concepts/dataflow-templates). This stage can also optionally anonymize or tokenize PII. When you need to handle workloads from different teams, if strong role separation is needed between them, we suggests to customize the scirpt and have separate `Tranformation` projects.
```
Comment (ludoo) [inline blueprints/data-solutions/data-platform-foundations/README.md:48]: same as above (trailing s)

**Review by ludoo — APPROVED** [review]
a few minor nits

File: blueprints/data-solutions/data-platform-foundations/IAM.md [hunk 4]
```diff
@@ -2,97 +2,97 @@
 
 Legend: <code>+</code> additive, <code>•</code> conditional.
 
-## Project <i>cmn</i>
+## Project <i>lc-2301-cmn</i>
```
[the remark this answers is not in the document]
Comment (lcaggio) [reply blueprints/data-solutions/data-platform-foundations/IAM.md:5]: This is the prefix, replaced with `prefix`

[the remark this answers is not in the document]
Comment (ludoo) [reply blueprints/data-solutions/data-platform-foundations/IAM.md:5]: the tool to extract IAM has a `--prefix` arg to remove the prefix for you

[the remark this answers is not in the document]
Comment (lcaggio) [reply blueprints/data-solutions/data-platform-foundations/IAM.md:5]: nice! didn't know. I'll use it.

File: blueprints/data-solutions/data-platform-foundations/variables.tf [hunk 5]
```diff
@@ -22,14 +22,47 @@ variable "billing_account_id" {
...
-    env_variables   = map(string)
+    disable_deployment = optional(bool)
+    environment_size   = string
+    software_config = object({
+      airflow_config_overrides = optional(any)
+      pypi_packages            = optional(any)
+      env_variables            = optional(map(string))
+      image_version            = string
+    })
+    workloads_config = object({
```
[the remark this answers is not in the document]
Comment (lcaggio) [reply blueprints/data-solutions/data-platform-foundations/variables.tf]: done

**Pull request merged**